render-test/runner.cpp

```diff
@@ -562,12 +572,35 @@ bool TestRunner::runOperations(const std::string& key, TestMetadata& metadata) {
             const mbgl::JSValue* propertyValue = &operationArray[3];
             layer->setLayoutProperty(propertyName, propertyValue);
         }
-    // probeMemoryStart
+    } else if (operationArray[0].GetString() == fileSizeProbeOp) {
+        // probeFileSize
+        assert(operationArray.Size() >= 3u);
+        assert(operationArray[1].IsString());
+        assert(operationArray[2].IsString());
+
+        std::string mark = std::string(operationArray[1].GetString(), operationArray[1].GetStringLength());
```

Comment:
maybe worth adding helper function that will convert rapidjson string value to std::string